### ISSUE TITLE
Implement new resource "Azure RM Service Endpoint"

### ISSUE THREAD

**Issue by ThePhen** [microsoft/terraform-provider-azuredevops#161]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or "me too" comments, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### New or Affected Resource(s)

* azuredevops_XXXXX

### Potential Terraform Configuration

```hcl
# Copy-paste your Terraform configurations here - for large Terraform configs,
# please use a service like Dropbox and share a link to the ZIP file. For
# security, you can also encrypt the files using our GPG public key.
```

### References

* #0000

**Comment by mo-saeed** [comment]
Any updates here ?

**Comment by mikaelkrief** [comment]
@nmiodice I am interressed to implement this before the #172

**Comment by nmiodice** [comment]
@mikaelkrief I don't understand your comment -- can you elaborate on the blocker you identified during research?

**Comment by mikaelkrief** [comment]
> @mikaelkrief I don't understand your comment -- can you elaborate on the blocker you identified during research?

@nmiodice : sorry, forgot my previous comment, I will soon finish this feature.

**Comment by tmeckel** [comment]
@nmiodice for housekeeping: this issue can be closed because PR #225 has been merged!!!!! 😄
